app/controllers/v0/institutions_controller.rb

```diff
@@ -170,7 +170,7 @@ def embed(group_counts)
     end
 
     def approved_institutions
-      Institution.version(@version[:number]).where(approved: true)
+      Institution.version(@version[:number]).where(approved: true).where("campus_type != 'E' OR campus_type IS NULL")
```

Comment:
To be clear, you want to return Institution records that have any campus_type value other than 'E', including `null`.  Is that correct?

Reply:
Correct